DataModel.cs

```diff
@@ -0,0 +1,209 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Timers;
+
+namespace ParallelBuildsMonitor
+{
+    /// <summary>
+    /// Container to hold information about build for one project
+    /// </summary>
+    public struct BuildInfo
+    {
+        public BuildInfo(string n, long b, long e, bool s)
+        {
+            name = n;
+            begin = b;
+            end = e;
+            success = s;
+        }
+        public long begin;
+        public long end;
+        public string name;
+        public bool success;
+    }
+
+    /// <summary>
+    /// Container to holds all build statistics data
+    /// </summary>
+    class DataModel
+    {
+        #region Members And Properties
+
+        /// <summary>
+        /// Holds point in time when entire build started (Solution).
+        /// </summary>
+        public DateTime StartTime { get; private set; }
+        public Dictionary<string, DateTime> CurrentBuilds { get; private set; } = new Dictionary<string, DateTime>(); //TODO: Check if collection can be changed even if private
+        public List<BuildInfo> FinishedBuilds { get; private set; } = new List<BuildInfo>(); //TODO: Check if collection can be changed even if private
```

Comment:
Using collections as read only properties is more complicated, you should probably use ReadOnlyCollectionTemplate, example:
```suggestion
         private List <BuildInfo> _FinishedBuilds = new List<BuildInfo>();
         ReadOnlyCollection <BuildInfo> FinishedBuilds { get { _FinishedBuilds.AsReadOnly(); } } 
```

Reply:
Should I rework that all members in all classes should begin from underscore (_)?
Right now all properties are Camel case, but members format is inconsistent (lowercase or Camel case)